docs/src/stream.rst

```diff
@@ -193,7 +193,7 @@ API
     initialized with `ipc` == 1.
 
     .. note::
-        `send_handle` must be a TCP socket or pipe, which is a server or a connection (listening
+        `send_handle` must be a TCP socket, or pipe on UNIX but not Windows, which is a server or a connection (listening
```

Comment:
It appears it can also be UDP, so perhaps this needs a more complete rewording

style nit: please wrap at 80-character line limit

Reply:
Done, thank you!